Sources/NIOHTTP1/NIOTypedHTTPClientUpgradeHandler.swift

```diff
@@ -191,6 +191,10 @@ public final class NIOTypedHTTPClientUpgradeHandler<UpgradeResult: Sendable>: Ch
         }
     }
 
+    public func errorCaught(context: ChannelHandlerContext, error: any Error) {
```

Comment:
We should probably fire the error on as well, to avoid breaking any systems that were relying on this.

Reply:
done